engine/wscript

```diff
@@ -23,7 +23,10 @@ def options(opt):
 		help = 'enable custom swap allocator. For devices with no swap support')
 
 	grp.add_option('--enable-legacy-sdl', action = 'store_true', dest = 'SDL12', default = False,
-		help = 'enable using SDL1.2 instead of SDL2(not recommended) [default: %default')
+		help = 'enable using SDL1.2 instead of SDL2(not recommended) [default: %default]')
+
+	grp.add_option('--enable-static-binary', action = 'store_true', dest = 'STATIC', default = False,
+		help = 'build static binary(not recommended, --single-binary required) [default: %default]')
```

Comment:
Why not to force activate single-binary if `--enable-static-binary` is selected?